lib/bundler_helper.rb

```diff
@@ -19,6 +19,7 @@ def self.require_groups(*groups)
             exit 1
           end
         end
+        Bundler.ui = Bundler::UI::Silent.new
```

Comment:
Why do we silence only on the `BundlerExt` environments?
Do I get this correctly this is only RedHat family production by default?

Reply:
https://github.com/theforeman/smart-proxy/pull/801#discussion_r712922320

Reply:
>  Do I get this correctly this is only RedHat family production by default?

https://github.com/theforeman/foreman-packaging/blob/58266ab62271af0bd8be57092868577e8a0212b9/debian/buster/foreman-proxy/control#L15

ruby-bundler-ext is in there so I think if bundler_ext is present, we can assume it's production